### DISCUSSION TITLE
Different impact on information risks

### DISCUSSION THREAD

**Discussion by DrRavik90** [monarc-project/MonarcAppFO#504] [Ideas]
Hi there,

I noticed that if you set up the CIA impacts on information risks it will automatically updated for all the risks associated with that asset. 

I think that it should not be like this. For ex:  The impact of CIA for IT Room is different in case of Loss of power supply and Failure of air-conditioning.

Does the tool allow different setup of the impact since i noticed that it does not. 

Thank You

**Comment by ruslanbaidan** [comment]
The impacts are propagated from the parent (primary or secondary) to its children assets. If needed, the impacts for secondary assets can be changed. If the values are inherited, there is a label about that in braces, e.g `Integrity : 3 (inherited)`
